docs/about/contributing.md

```diff
@@ -30,19 +30,16 @@ If you are familiar with GitHub please either:
 If you are submitting a pull request:
 
 - Please create one pull request per use case so admin can check the complete change
-
 - Request review from @Acharbonneau and @marisalim
-
 - Please allow up to one week for admin to review your request
 
 ### Use Case approval process
 - The Use Case committee will mark proposed use cases as `in progress`, `approved`, and `done`, as appropriate.
-  
 - If the Use Case has been approved, a &#x2705; will be added next to the Use Case tab title, as well as its corresponding Requirements tab titles.
```

Comment:
A checkmark means done, not approved. We need to find approved and in progress insignia

Reply:
Ah ok, will edit!

Reply:
how about this?
:hourglass: for in progress
:ok_hand: for approved

Reply:
Works for me